modules/ROOT/pages/getting-started.adoc

```diff
@@ -80,6 +80,19 @@ Find the `@Extension` annotated class and
  examine <<module-structure#, module structure>> and the different
 elements that you can add to a module. Use these elements to modify a connector.
 
+[[customicon]]
+== Add a Custom Icon to Your Module
+
+Module icons are files in the SVG format.
+
+To customize the icon of a connector, add an icon.svg file that contains the custom icon under the icon folder in the root of the connector tree.
```

Comment:
```suggestion
To customize the icon of a connector, add an `icon.svg` file that contains the custom icon under the `icon` folder in the root of the connector tree.
```

Reply:
Is icon.svg an actual name? Or is it supposed to be a variable (ie, the user can name the icon whatever they want) like <icon-name>.svg?

Reply:
@valkyrie69 It seems that the icon file name must be icon.svg. As far as I can tell, any other file name or format will be ignored. So, if you don't have an icon.svg file under the icon directory, Anypoint Studio and Exchange won't be able to show the custom icon.

Reply:
Yeah, it is an actual name, so it has to be named icon.svg.